test_psa_target.py

```diff
@@ -126,13 +126,15 @@ def _run_regression_test(args):
     Run TF-M regression test for the target
     :param args: Command-line arguments
     """
-    logging.info("Test TF-M regression...")
+    logging.info("Build TF-M regression tests for %s" % args.mcu)
```

Comment:
```suggestion
    logging.info("Build TF-M regression tests for %s", args.mcu)
```

So we don't waste time formatting the string before it gets passed to the logger.